.circleci/config.yml

```diff
@@ -0,0 +1,93 @@
+version: 2.1
+
+default_version: &default_version 1.8.1
+
+jobs:
+  build:
+    parameters:
+      version:
+        description: Elixir Version
+        type: string
+        default: *default_version
+
+    parallelism: 1
+    docker:
+      - image: elixir:<< parameters.version >>
+
+    working_directory: ~/app
+
+    steps:
+      - checkout  # check out source code to working directory
+      - run: wget https://github.com/nats-io/gnatsd/releases/download/v1.4.1/gnatsd-v1.4.1-linux-amd64.zip
```

Comment:
Do you want to keep this locked down to a specific version or would you rather run a docker service like `nats:latest`?

Reply:
I looked at using a docker container, but starting 4 of them with different settings was a bit awkward and I like the idea of picking a specific version to test against. I think once 2.0 is released we'll probably test against 2 versions at a time.